lib/cli/src/commands/run/wasi.rs

```diff
@@ -49,11 +49,6 @@ pub struct Wasi {
     #[clap(long = "use", name = "USE")]
     uses: Vec<String>,
 
-    /// List of webc packages that are explicitely included for execution
-    /// Note: these packages will be used instead of those in the registry
-    #[clap(long = "include-webc", name = "WEBC")]
-    include_webcs: Vec<String>,
```

Comment:
The removal of this parameter needs to be replaced with another way of making the integration tests deterministic or it will prevent them from properly regression testing our changes and will require the availability of registries in order for the tests to run successfully.

One way is to create a dummy local registry that the unit tests can point to which is probably the better way to do it, either way we shouldn't remove this code until the replacement is in place.

Reply:
Good point. I'm guessing I'll need to wire that into the `BuiltinResolver` somehow and then make sure `wasmer run` configures it's own `BuiltinResolver` and passes to to the `PluggableRuntime`.